utils/unittest/fixtures.go

```diff
@@ -991,7 +991,7 @@ func IdentityFixture(opts ...func(*flow.Identity)) *flow.Identity {
 	stakingKey := StakingPrivKeyByIdentifier(nodeID)
 	identity := flow.Identity{
 		NodeID:        nodeID,
-		Address:       fmt.Sprintf("address-%x", nodeID[0:7]),
+		Address:       fmt.Sprintf("address-%x:1234", nodeID[0:7]),
```

Comment:
If we need to set a port in our fixtures, please always set it to zero. A zero port has a special meaning: it tells the OS to replace it with an available port number. As this `IdentityFixture` is used widespread, it allows running a networking instance on each generated identity without the concern of conflicting ports. 
```suggestion
		Address:       fmt.Sprintf("address-%x:0", nodeID[0:7]),
```

Reply:
https://github.com/onflow/flow-go/pull/2990/commits/adf179e566af311f630cd8b1d5137541413bb672